.github/scripts/amqa/amqa_metrics.py

```
@@ -0,0 +1,41 @@
+def main() -> int:
+    plan_path = Path(os.environ.get("AMQA_PLAN_PATH", "/tmp/amqa/qa-plan.json"))
+    result_path = Path(os.environ.get("AMQA_RESULT_PATH", "/tmp/amqa/qa-result.json"))
+
+    plan = json.loads(plan_path.read_text()) if plan_path.is_file() else {}
+    result = json.loads(result_path.read_text()) if result_path.is_file() else {}
+
+    summary_path = os.environ.get("GITHUB_STEP_SUMMARY", "")
+    lines = [
+        "## AMQA Metrics",
+        "",
+        f"| Metric | Value |",
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)